src/messaging/messaging.ts

```diff
@@ -41,6 +41,8 @@ const CAMELCASED_NOTIFICATION_PAYLOAD_KEYS_MAP = {
   clickAction: 'click_action',
   titleLocArgs: 'title_loc_args',
   titleLocKey: 'title_loc_key',
+  subtitleLocArgs: 'subtitle_loc_args',
+  subtitleLocKey: 'subtitle_loc_key',
```

Comment:
This is for the legacy send operations. I don't think we should be adding new features to this API. Lets revert these two lines.

Reply:
ok done